assets/src/scss/base/_colors.scss

```diff
@@ -40,9 +40,11 @@ $active-blue:  #05324c;
 $peach:       #eaccbb;
 
 // Social buttons
+$share-buttons-bg-color: #f6f4ef;
```

Comment:
You can now use `tokens` from [here](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_tokens.scss#L37-L38)